pkg/artifactory/provider.go

```diff
@@ -100,16 +100,18 @@ func Provider() *schema.Provider {
 			"artifactory_group":                      resourceArtifactoryGroup(),
 			"artifactory_user":                       resourceArtifactoryUser(),
 			"artifactory_permission_target":          resourceArtifactoryPermissionTarget(),
-			"artifactory_replication_config":         resourceArtifactoryReplicationConfig(),
```

Comment:
We need to retain these existing resources for backward compatibility. Deprecation message should be added to these 2 resources so we can give users warning on them being replaced by the new resources.

Reply:
See lines 112-113

Reply:
Got it. We still need to add deprecation messages to the existing resources.